plugins/inputs/netflow/type_conversion.go

```diff
@@ -0,0 +1,619 @@
+package netflow
```

Comment:
This file is missing a bit of test coverage. While I do not think you need to hit every single switch case, I do think it would be good to fully cover the `decode*` functions where possible, specifically:

* decodeInt32
* decodeBool
* decodeString
* decodeMAC
* decodeIPv4Options
* decodeFragmentFlags

And some additonal path coverage of

* mapL4Proto
* decodeTCPFlags